bindgen-tests/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "bindgen-tests"
+edition = "2018"
+version = "0.1.0"
+
+include = [
+  "Cargo.toml",
+  "build.rs",
+]
+
+[dev-dependencies]
+bindgen = { version = "=0.60.1", path = "../bindgen" }
```

Comment:
Do we really need the explicit version? That means that bumping bindgen would also need bumping this which seems unfortunate.